app/assets/javascripts/api.jsx

```diff
@@ -0,0 +1,59 @@
+
+
+class Api {
+
+  static get(path, data = null) {
+    return Api.ajax(path, 'GET', data);
+  }
+  static post(path, data = null) {
+    return Api.ajax(path, 'POST', data);
+  }
+  static put(path, data = null) {
+    return Api.ajax(path, 'PUT', data);
+  }
+
+   static delete(path, data = null) {
+        return Api.ajax(path, 'DELETE', data);
+   }
+  static ajax(path, method = 'GET', data = null) {
+      return $.ajax({
+      url: config.api + path,
+      method: method,
+      contentType: 'application/json',
+      data: JSON.stringify(data)
+    });
+  }
+
+    static getRestaurants(dispatch = store.dispatch) {
+        Api.get("/restaurants.json")
+            .done(restaurants => dispatch(RestaurantsActions.loaded(restaurants)))
+            .fail(error => {
+                const message = error.message || error;
+                dispatch(RestaurantsActions.failedToLoad(message));
+            });
+    }
+
+    static createRestaurant(name, food_type, speed, ten_bis, delivery, rank, link, description, address, lat, lang, dispatch = store.dispatch) {
```

Comment:
This will require to change method if you'll want to add field and very vulnerable to typos. Maybe better to pass object instead of arguments.